src/main/java/ru/skypro/homework/mappers/UserMapper.java

```diff
@@ -41,9 +41,9 @@ public UserDTO userToUserDTO(User user) {
         dto.setId(user.getId());
         dto.setFirstName(user.getFirstName());
         dto.setLastName(user.getLastName());
+        dto.setImage("/images/get/" + user.getImage().getId());
```

Comment:
```suggestion
        if(user.getImage()!=null){
            dto.setImage("/images/get/" + user.getImage().getId());
        }
```
Добавьте проверку на null